Sources/FoundationExtensions.swift

```diff
@@ -24,29 +26,33 @@ extension NotificationCenter {
 	///         will terminate immediately with an `interrupted` event.
 	///         Otherwise, the producer will not terminate naturally, so it must
 	///         be explicitly disposed to avoid leaks.
-	public func rac_notifications(forName name: Notification.Name?, object: AnyObject? = nil) -> SignalProducer<Notification, NoError> {
+	public func notifications(forName name: Notification.Name?, object: AnyObject? = nil) -> SignalProducer<Notification, NoError> {
 		// We're weakly capturing an optional reference here, which makes destructuring awkward.
 		let objectWasNil = (object == nil)
-		return SignalProducer { [weak object] observer, disposable in
+		return SignalProducer { [base, weak object] observer, disposable in
```

Comment:
Can you use `self.base` here to be explicit please? It took me a while to figure out where this value was coming from, because it's not in the current context.